.drone.yml

```diff
@@ -231,6 +231,24 @@ steps:
     - drone-publish.rancher.io
     status:
     - failure
+- name: docker-publish-rancherd
```

Comment:
probably should do this before slack notify

Reply:
moved